examples/lusid/holdings/upload-holding.sql

```diff
@@ -0,0 +1,31 @@
+-- ============================================================
+
+-- Description:
+-- 1. In this query, we upload a holding into LUSID.
+-- 2. First, we specify the portfolio code and scope and date.
+-- 3. Next, we create a table for the holding data.
+-- 4. Finally we upload the holding data into LUSID.
+
+-- ============================================================
+
+-- Set variables for the portfolio's scope and code
+@@portfolio_scope =
+select 'IBOR';
+
+@@portfolio_code =
+select 'uk-equity';
+
+-- Set variable for the current date
+@@today =
+select date ('now');
+
+-- Create table for an equity holding 
+@holding_data =
```

Comment:
This is correct but ideally we would want to read holdings data from a file in drive. Are you able to create a dummy holdings file in drive and then read that in?